Trim white space, compare and update sale rates.

chuckbucket opened the pull request:
1: White space
   Brick Owl appears to always trim white space from comments and remarks. Bricklink does not. This would cause Bricksync to always try and update a remark or comment when comparing "C-02" and "C-02   ".  Bricksync will now see them as the same and not try to update the lot.

2: Compare and update sales rates.
  Bricksync will now look at the sales rates and make updates as needed. This was previously excluded and would not be updated during syncing.

ZZJHONS commented:
Wouldn't be better if the space would actually be trimmed from Bricklink instead?

Also having sales not sync was by design as some stores prefer to run sales independently, but maybe we can add this with a setting in the config file?

Thanks for your work!

chuckbucket commented:
@ZZJHONS, I believe that I addressed your comments with the latest two commits to the. I have been running a version with my changes are it seams to be running without issue.
